### ISSUE TITLE
Unable to find a specification for `RWMRecurrenceRule (~> 0.0.2)`

### ISSUE THREAD

**Issue by agordeev** [rmaddy/RWMRecurrenceRule#1]
Did you do `pod trunk push`?

**Comment by rmaddy** [comment]
Do you mean https://github.com/rmaddy/RWMRecurrenceRule/blob/master/RWMRecurrenceRule.podspec

**Comment by agordeev** [comment]
Right, but currently the pod is not listed on Cocoapods and therefore cannot be installed via `pod install`.

Looks like you forgot to do `pod trunk push` as stated here: https://guides.cocoapods.org/making/making-a-cocoapod.html

**Comment by johnsusek** [comment]
I was able to do this to install,
`  pod 'RWMRecurrenceRule', :git => 'https://github.com/rmaddy/RWMRecurrenceRule.git'`
